packages/@lwc/module-resolver/src/utils.ts

```diff
@@ -7,6 +7,8 @@
 import path from 'path';
 import fs from 'fs';
 
+import { isObject } from '@lwc/shared';
```

Comment:
I would rather implement the isObject rather than bringing the whole package for this one-off